src/app/components/MenueLink/MenueLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import type { ReactNode } from 'react';
+import Typography from '../Typography/Typography';
+import styles from './MenueLink.module.css';
+
+export type MenueLinkProps = {
+  children: ReactNode;
+  to: string;
+  className?: string;
+};
+
+function MenueLink({ children, to, className }: MenueLinkProps): JSX.Element {
```

Comment:
🧐 I believe 'menu' is spelled incorrectly

Reply:
Oops... 